src/Homework6/task2.py

```diff
@@ -0,0 +1,41 @@
+"""Создайте декоратор, который вызывает задекорированную функцию пока
+   она не выполнится без исключений (но не более n раз - параметр декоратора).
+   Если превышено количество попыток, должно быть возбуждено исключение
+   типа TooManyErrors
+"""
+
+
+class TooManyErrors(Exception):
+    print("TooManyErrors: the function is called more times.")
+
+
+# Декоратор для основной функции
+def decorator_function(func, try_count=5):
+    func.__count_run__ = 0  # добавили переменную в функцию
```

Comment:
this function would be called one time, exceed it's limit and it wouldn't be able to be run any more